puppet/zulip/files/hooks/pre-deploy.d/sentry.hook

```diff
@@ -51,9 +62,26 @@ echo "$SENTRY_RELEASE" >./sentry-release
 echo "sentry: Creating release $SENTRY_RELEASE"
 
 export SENTRY_AUTH_TOKEN
-sentry-cli releases --org="$SENTRY_ORG" --project="$SENTRY_PROJECT" new "$SENTRY_RELEASE"
+
+# sentry-cli only supports passing one project when making a new
+# release, and we want to possibly create more than once at once.  Use
+# curl to make the API request.
```

Comment:
Not important but I suppose we could open an issue with them about this, if there isn't one already.

Reply:
https://github.com/getsentry/sentry-cli/issues/734 and it was closed as WONTFIX, essentially.